### PR TITLE
meta tags added

### LINKED ISSUE #31: SEOs
Add various meta tags required for it boost up the rating using lighthouse, pageinsights

### PR DESCRIPTION
**Pull request by maanpatel2492**
Fixes #31 

I have added meta tags and description and SEO according to the lighthouse in more than 90%.
1) Please check the description and title once and give feedback accordingly.If there is any mistake I will rework on it.
2) In authors column in meta tags does I have to write all the team members name?

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+    <!doctype html>
```
Comment (harikesh409) [inline team.html:1]: remove the unnecessary space.

File: team.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 <html lang="en">
 
 <head>
```
Comment (harikesh409) [inline team.html:4]: add the meta tags in `team.html` also.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harikesh409** [comment]
@maanpatel2492 resolve the conflicts.

File: team.html (hunk 2 above)
Comment (maanpatel2492) [reply team.html:4]: Okay

**Comment by ravikr126** [comment]
is this completed @maanpatel2492

**Comment by maanpatel2492** [comment]
> is this completed @maanpatel2492

yes it is.

**Pull request closed without merging**